osf/models/metaschema.py

```diff
@@ -158,19 +159,16 @@ def absolute_api_v2_url(self):
 
 class RegistrationFormBlock(ObjectIDMixin, BaseModel):
     class Meta:
-        unique_together = ('schema', 'block_id')
         order_with_respect_to = 'schema'
 
     schema = models.ForeignKey('RegistrationSchema', related_name='form_blocks', on_delete=models.CASCADE)
-    page = models.CharField(max_length=255)
-    section = models.CharField(max_length=255, null=True)
     help_text = models.TextField()
-    block_id = models.CharField(max_length=255, db_index=True)
+    example_text = models.TextField(null=True)
```

Comment:
Added `example_text` for prereg challenge and osf preregistration.  For these schemas, `description` was clearly `help_text` and there was no place, then, for the `help_text` to go.  (Neither one should be displayed to the user, and merging them together would be too lengthy.  Alternatively, we could just discard the original help_text for these two schemas.

help -> example_text
description -> help_text

Reply:
question for product -